src/lib/graphqlErrorHandler.js

```diff
@@ -20,12 +20,23 @@ export const shouldReportError = originalError => {
   return true
 }
 
+const isStaging = process.env.GRAVITY_API_URL.includes("staging")
+const subdomain = isStaging ? "metaphysics-staging" : "metaphysics-production"
```

Comment:
It’s generally better to get the domain from the request rather than hardcoding any.